### PR TITLE
Update osx install instructions

### PR DESCRIPTION
**Pull request by will-moore**
Additional work on top of ongoing #1766.

Should address most of the comments on that PR, as well as a few other fixes noticed on the way.

 - Told users to download OMERO.server.. from downloads page, not putting version number in docs since this will be out of date very soon.
 - Addressed path issues in a couple of ways.

### CODE REVIEW

File: omero/sysadmins/unix/server-install-homebrew.rst [hunk 1]
```diff
@@ -41,175 +52,262 @@ running::
...
+    $ virtualenv ~/Virtual/omero
+
+   This will create a folder to hold Python libraries in the the directory :file:`~/Virtual/omero/lib`
+
+  .. note:: 
+   You can activate the Virtualenv environment that we created using::
+
+    $ source ~/Virtual/omero/bin/activate
+
+   This will swtich to using Pip and Python in the Virtualenv directory
```
Comment (mtbc) [inline omero/sysadmins/unix/server-install-homebrew.rst]: switch

File: omero/sysadmins/unix/server-install-homebrew.rst [hunk 2]
```diff
@@ -41,175 +52,262 @@ running::
...
+   Find the current OMERO.server zip from the
+   `downloads page <https://downloads.openmicroscopy.org/latest/omero/artifacts/>`_.
+   Download and extract the OMERO.server-x.x.x-ice36-bxx.zip.
+
+2. Once extracted, open your :file:`.bash_profile` in a text editor, 
+   such as the built-in TextEdit app::
+
+    $ open -a TextEdit.app ~/.bash_profile
+
+   Add an environment variable :envvar:OMERO_SERVER to the :file:`.bash_profile` which points
```
Comment (mtbc) [inline omero/sysadmins/unix/server-install-homebrew.rst]: need the backticks

File: omero/sysadmins/unix/server-install-homebrew.rst [hunk 3]
```diff
@@ -41,175 +52,262 @@ running::
...
-OMERO dependencies::
+  .. seealso::
+   :doc:`/developers/installation`
+        Developer documentation page on how to check out to source code
+   :doc:`/developers/build-system`
+        Developer documentation page on how to build the OMERO.server
 
-    $ brew install --only-dependencies omero
+5. Once the build completes, the OMERO server build output will be located in :file:`~/Projects/Omero/code/openmicroscopy/dist`.
+   If it is not already open, open your :file:`.bash_profie`::
```
Comment (mtbc) [inline omero/sysadmins/unix/server-install-homebrew.rst]: profile

File: omero/sysadmins/unix/server-install-homebrew.rst [hunk 4]
```diff
@@ -41,175 +52,262 @@ running::
...
 
-Then prepend the development :file:`bin` directory to your :envvar:`PATH` to
-pick the right executbale::
+   Reload :file:`.bash_profile` in OS X::
 
-    $ export PATH=~/code/projects/openmicroscopy/dist/bin:$PATH
+    $ source ~/.bash_profile
 
-and follow the steps for setting up the database and OMERO data directory as mentioned in the previous section.
+2. To use Omero, we need to first setup Postgres. Open a command-line terminal and run the
```
Comment (mtbc) [inline omero/sysadmins/unix/server-install-homebrew.rst]: "set up"

File: omero/sysadmins/unix/server-install-homebrew.rst [hunk 5]
```diff
@@ -4,8 +4,9 @@ OMERO.server installation on OS X with Homebrew
 .. topic:: Overview
 
     This walkthrough demonstrates how to install OMERO on a clean Mac
-    OS X system (10.9 or later) using Homebrew.  Note that this
-    demonstrates how to install OMERO.server *from the source code*
+    OS X system (10.9 or later) using Homebrew.  This demonstrates how to install
+    OMERO.server, either from the downloaded OMERO.server.zip
+    or *from the source code*
```
Comment (pwalczysko) [inline omero/sysadmins/unix/server-install-homebrew.rst]: Did you mean to mark it up as italics ? Maybe bold would be better (two stars)

Comment (will-moore) [reply omero/sysadmins/unix/server-install-homebrew.rst]: Don't think it needs bold or italics really.

File: omero/sysadmins/unix/server-install-homebrew.rst (hunk 2 above)
Comment (pwalczysko) [reply omero/sysadmins/unix/server-install-homebrew.rst]: ditto, the :envvar:OMERO_SERVER does not build correctly

File: omero/sysadmins/unix/server-install-homebrew.rst (hunk 5 above)
Comment (pwalczysko) [reply omero/sysadmins/unix/server-install-homebrew.rst]: agree, plain is the best

**Comment by will-moore** [comment]
Minor fixes pushed.

File: omero/sysadmins/unix/server-install-homebrew.rst [hunk 6]
```diff
@@ -177,28 +168,24 @@ Pre-built server
...
 
     # OMERO Server distribution directory
-    export OMERO_SERVER=~/Projects/Omero/server
+    export OMERO_SERVER=/path/to/OMERO.server-x.x.x-ice36-bxx
 
-   and add the OMERO executabale to the OS X :envvar:`PATH`::
+   and add the OMERO executable to the OS X :envvar:`PATH`::
 
     # Add the OMERO distribution to PATH
     export PATH=OMERO_SERVER/bin:$PATH
```
Comment (pwalczysko) [inline omero/sysadmins/unix/server-install-homebrew.rst]: ``export PATH=$OMERO_SERVER/bin:$PATH`` works (missing dollar sign)

**Comment by will-moore** [comment]
Fixed export PATH=$OMERO_SERVER/bin

File: omero/sysadmins/unix/server-install-homebrew.rst [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
 .. topic:: Overview
 
     This walkthrough demonstrates how to install OMERO on a clean Mac
-    OS X system (10.9 or later) using Homebrew.  Note that this
-    demonstrates how to install OMERO.server *from the source code*
+    OS X system (10.9 or later) using Homebrew.  This demonstrates how to install
+    OMERO.server, either from the downloaded OMERO.server.zip
+    or from the source code
     via Homebrew, in addition to all its prerequisites. It is aimed at **developers**
     since typically MacOS X is not suited for serious server deployment.
```
Comment (pwalczysko) [inline omero/sysadmins/unix/server-install-homebrew.rst:12]: @jburel recommends removing the reference to the scripts completely - no guarantee these work @will-moore  (see the sentence below this comment ``These instructions....``

**Comment by will-moore** [comment]
I've removed the paragraph mentioning the install scripts.

**Comment by pwalczysko** [comment]
Reads well. The installation works.

Ready to merge FMPOV.

**Comment by will-moore** [comment]
As I just noticed: https://github.com/openmicroscopy/ome-documentation/pull/1766#discussion_r141197315 we don't seem to have docs on how to run nginx on Mac. ```sysadmins/unix/install-web.html``` is not an install page but "OMERO.web administration".

**Pull request merged**